__tests__/app.test.js

```diff
@@ -291,3 +291,32 @@ describe("/api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  describe("DELETE", () => {
+    test("204: responds with empty body and deletes comment from database", () => {
+      return request(app)
+        .delete("/api/comments/5")
+        .expect(204)
+        .then(({ body }) => {
+          expect(body).toEqual({});
```

Comment:
By default Express won't send back anything whenever we send status 204, so this line checking the body is empty isn't entirely necessary. 
The status 204 that we assert with Supertest is sufficient.